### PR TITLE
[Filebeat][New Fileset] Cisco Umbrella support

### LINKED ISSUE #20694: Cisco Umbrella Filebeat Module
## Filebeat module
Module: Cisco Umbrella
Documentation: https://docs.umbrella.com/deployment-umbrella/docs/log-formats-and-versioning

Looking to get the DNS, Proxy, IP and Cloud Firewall logs parsed to ECS format. 

* [X] Test log files exist for the grok patterns (Exist in documentation link)
* [X] Generated output for at least 1 log file exists (Exist in documentation link)

### PR DESCRIPTION
**Pull request by P1llus**
## What does this PR do?

This adds support for Cisco Umbrella logs to filebeat

## Why is it important?

Adds support for new OOTB integrations for filebeat

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

- Closes elastic/beats#20694

### CODE REVIEW

**Comment by P1llus** [comment]
jenkins test this please

**Review by marc-gr — APPROVED** [review]

File: filebeat/docs/modules/cisco.asciidoc [hunk 1]
```diff
@@ -379,6 +383,59 @@ will be found under `rsa.raw`. The default is false.
...
+
+To configure Cisco Umbrella to log to either your own S3 bucket or one that is managed by Cisco please follow the https://docs.umbrella.com/deployment-umbrella/docs/log-management[Cisco Umbrella User Guide]
+
+This fileset supports all 4 log types which is:
+- Proxy
+- Cloud Firewall
+- IP Logs
+- DNS logs
+
+The Cisco Umbrella fileset is dependant on that the original file path structure is to be followed. This structure is documented https://docs.umbrella.com/deployment-umbrella/docs/log-formats-and-versioning[Umbrella Log Formats and Versioning]:
```
Comment (adriansr) [inline filebeat/docs/modules/cisco.asciidoc:399]: First phrase sounds a little odd to me. What about:
```suggestion
The Cisco Umbrella fileset depends on the original file path structure being followed. This structure is documented https://docs.umbrella.com/deployment-umbrella/docs/log-formats-and-versioning[Umbrella Log Formats and Versioning]:
```

File: filebeat/docs/modules/cisco.asciidoc [hunk 2]
```diff
@@ -379,6 +383,59 @@ will be found under `rsa.raw`. The default is false.
...
+
+<subfolder>/<YYYY>-<MM>-<DD>/<YYYY>-<MM>-<DD>-<hh>-<mm>-<xxxx>.csv.gz
+dnslogs/<year>-<month>-<day>/<year>-<month>-<day>-<hour>-<minute>.csv.gz
+
+When configuring the fileset, please ensure that the Queue url is set to the root folder that includes each of these subfolders above.
```
Comment (adriansr) [inline filebeat/docs/modules/cisco.asciidoc:404]:
```suggestion
When configuring the fileset, please ensure that the Queue URL is set to the root folder that includes each of these subfolders above.
```

File: filebeat/docs/modules/cisco.asciidoc [hunk 3]
```diff
@@ -10,13 +10,15 @@ This file is generated! See scripts/docs_collector.py
...
 
-This is a module for Cisco network device's logs. It includes the following
+This is a module for Cisco network device's logs and Cisco Umbrella. It includes the following
 filesets for receiving logs over syslog or read from a file:
 
 - `asa` fileset: supports Cisco ASA firewall logs.
 - `ftd` fileset: supports Cisco Firepower Threat Defense logs.
 - `ios` fileset: supports Cisco IOS router and switch logs.
 - `nexus` fileset: supports Cisco Nexus switch logs.
+- `meraki` fileset: supports Cisco Meraki logs.
```
Comment (adriansr) [inline filebeat/docs/modules/cisco.asciidoc:20]: Thanks for adding this missing entry.

**Review by adriansr — CHANGES_REQUESTED** [review]
Just some suggestions for docs

**Comment by P1llus** [comment]
Thanks for the pointers @adriansr, the doc changes totally made sense! Added them in :)

**Review by adriansr — APPROVED** [review]

**Pull request merged**